docs/photon/codegen-and-node-setup.md

```diff
@@ -35,15 +35,15 @@ node_modules/@generated/photon
 
 ## Photon.js should be viewed as an npm package
 
-Node.js libraries are typically installed as npm dependencies using `npm install`. The respective packages are then located inside the [`node_modules`](https://docs.npmjs.com/files/folders#node-modules) directory from where they can be easily imported into application code.
+Node.js libraries are typically installed as npm dependencies using `npm install`. The respective packages are then located inside the [`node_modules`](https://docs.npmjs.com/files/folders#node-modules) directory from where they can be imported into application code.
 
 Because Photon.js is a custom API for _your_ specific database setup, it can't follow that model. It needs to be generated locally instead of being installed from a central repository like npm. However, the mental model for Photon.js should still be that of an Node module.
 
 ## Why is Photon.js generated into `node_modules/@generated` by default?
 
-### Easy imports
+### One-line imports
```

Comment:
```suggestion
### Non-relative imports
```


One-line import isn’t the correct term here as all imports in JS/TS are one liner.

Reply:
Is any qualifier for "imports" needed here? 
`Importing Photon.js` could work as well.

Reply:
Totally agree!

Reply:
I also agree with @janpio 

```suggestion
### Importing Photon.js
```